draft-ietf-tls-tls13.md

```diff
@@ -2322,6 +2340,13 @@ secp256r1, etc.
   X9.62 {{X962}} and FIPS 186-4 {{DSS}}.
   Values 0xFE00 through 0xFEFF are reserved for private use.
 
+x25519, etc.
```

Comment:
Since there are only two in the list, I think that listing both is better than using "etc"